coordinator/quote/ert.go

```diff
@@ -26,6 +26,8 @@ type PackageProperties struct {
 	ProductID *uint64
 	// Security version number of the package.
 	SecurityVersion *uint
+	// Accepted TCB levels
+	AcceptedTcbLevels []string
```

Comment:
```suggestion
	AcceptedTCBLevels []string
```
Just a small naming thing